src/components/editor/property/InputLiteral.jsx

```diff
@@ -91,17 +91,7 @@ export class InputLiteral extends Component {
   }
 
   handleDeleteClick = (event) => {
-    const labelToRemove = event.target.dataset.content
-    const idToRemove = event.target.dataset.item
-
-    this.props.handleRemoveItem(
-      {
-        id: idToRemove,
-        label: labelToRemove,
-        reduxPath: this.props.reduxPath,
-        uri: this.props.propertyTemplate.propertyURI,
```

Comment:
why were label and uri here before?  were they never used? 

Putting it another way -- we had no test anywhere showing why/how these were used?  If so, please write a ticket to write such a test (showing functionality of the delete, prolly as an integration test)

Reply:
They were never used. That's why I got rid of them, because it's confusing that we're suppling them.

Reply:
I think initially we searched based on label of the item. This is no longer needed in PR.